File: src/main/java/graphicscore/player/ScorePlayerManager.java
package graphicscore.player;

import cn.nukkit.Player;
import graphicscore.GraphicScore;
import java.util.UUID;
import java.util.HashMap;
import java.util.Map;

public class ScorePlayerManager {
    private static final Map<UUID, ScorePlayer> playerMap = new HashMap<>();

    public static ScorePlayer addPlayer(Player player) {
        ScorePlayer session = new ScorePlayer(player, GraphicScore.getDefaultBoard());
        playerMap.put(player.getUniqueId(), session);
        return session;
    }

    public static ScorePlayer getPlayer(Player player) {
        return playerMap.get(player.getUniqueId());
    }

    public static void removePlayer(Player player) {
        playerMap.remove(player.getUniqueId());
    }
}

File: src/main/kotlin/me/skh6075/pnx/graphicscore/player/ScorePlayerManager.kt
package me.skh6075.pnx.graphicscore.player

import cn.nukkit.Player
import me.skh6075.pnx.graphicscore.GraphicScore
import java.util.UUID

object ScorePlayerManager {
    private val playerMap: MutableMap<UUID, ScorePlayer> = mutableMapOf()

    fun addPlayer(player: Player) : ScorePlayer {
        val session = ScorePlayer(player, GraphicScore.defaultBoard)
        playerMap[player.uniqueId] = session

        return session
    }

    fun getPlayer(player: Player) : ScorePlayer? {
        return playerMap[player.uniqueId]
    }

    fun removePlayer(player: Player) {
        playerMap.remove(player.uniqueId)
    }
}